sympy/vector/parametricregion.py

```diff
@@ -0,0 +1,48 @@
+from sympy.core.basic import Basic
+
+class ParametricRegion(Basic):
+    """
+    Represents a parametric region in space.
+    """
+    def __init__(self, system, *args):
+        """
+        Examples
+        ========
+
+        >>> from sympy import symbols, cos, sin, pi
+        >>> from sympy.vector import CoordSys3D, ParametricRegion
+        >>> r, theta = symbols("r theta")
+        >>> C = CoordSys3D('C')
+        >>> circle = ParametricRegion(C, r*cos(theta), r*sin(theta), (theta, 0, 2*pi))
```

Comment:
Is the parametric region defined within a `CoordSys3D` object? Can this be made optional? It's easier to use if you don't have to define too many objects.

Reply:
Yes, It will be easier if a user does not have to pass the `CoordSys3D` object. But we need to determine base scalars otherwise the definition of a parametric region is incomplete. A parametric representation is just defining base scalars in terms of some parameters. 

We can avoid passing the `CoordSys3D` object if we use the base scalars of the vector/scalar field  which needs to be integrated. So, if a user does not pass a `CoordSys3D` object, then SymPy will use the base scalars of the vector/scalar field used for integration. 

```python
C = CoordSys3D('C')
R = C.locate_new('R', 3*C.i + 4*C.j + 5*C.k)
p = ParametricRegion(t, t**2, (t, 0, 1))
ParametricIntegral(C.x*C.y*C.z*C.i, p) # Define p in terms of C.x, C.y and C.z 
ParametricIntegral(R.x*R.y, p) # Define p in terms of R.x, R.y and R.z  
```

Reply:
What about using a convention similar to Lambda?

Reply:
@Upabjojr Can you please give an example.